docs/src/api.md

```diff
@@ -167,8 +221,74 @@ PulseDesigner.spiral_base
 PulseDesigner.EPI_example
 ```
 
+## Spin Representation
```

Comment:
Bad name. Please improve. `Mag` is the only `<:SpinRepresentationState`. `Rx`, `Ry`, `Rz`, and `Q` return `Spinor`'s. Not sure about `Un`.

Reply:
Move Mag to simulation. Remove Spinor-related exports.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/258/commits/11fa41ab96ff275ebf6c0d7152359435336dcf32

Reply:
Incorrect usage of import.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/258/commits/5427a79a4453a3e5fc1ff9702e2947010ff3b0db